Framework/PythonInterface/mantid/simpleapi.py

```diff
@@ -253,8 +253,14 @@ def StartLiveData(*args, **kwargs):
     except KeyError:
         pass
 
-    # Handle unpacking / workspace names from lhs
+    # LHS Handling currently unsupported for StartLiveData
     lhs = _kernel.funcinspect.lhs_info()
+    if lhs[0] > 0:  # Number of terms on the lhs
+        raise RuntimeError("Assigning the output of StartLiveData is currently "
```

Comment:
After talking to @martyngigg, we decided that for now this will have to be considered a limitation of the simpleapi. There are several issues with capturing the output:

1. Unwrapping is not working properly because of the optional "AccumulationWorkspace" output, and there's no easy fix for this.
1. Even capturing in a single variable (so capturing output as a tuple without unwrapping) is setting the AccumulationWorkspace argument to the name of the variable on the left.
1. There if the name of the first variable on the left matches the value of the "OutputWorkspace" argument, there is very inconsistent / buggy behaviour because now both OutputWorkspace and AccumulationWorkspace have the same name.
1. Since OutputWorkspace is being periodically replaced in the ADS, any reference to it you get in Python is invalidated shortly after you get it.